lib/action_subscriber.rb

```diff
@@ -81,6 +81,7 @@ def self.print_subscriptions
         logger.info "    --    exchange: #{route.exchange}"
         logger.info "    --       queue: #{route.queue}"
         logger.info "    -- routing_key: #{route.routing_key}"
+        logger.info "    --  threadpool: #{route.threadpool.pool_size}"
```

Comment:
I would like to make a cut at adding the threadpool name here, but I can't do it without some hacky code.